src/loader.rs

```diff
@@ -39,6 +39,16 @@ impl<P> RegisterContext<P> {
         self.info.actions = Some(entrypoints::actions);
         self
     }
+
+    pub fn with_password(mut self) -> Self {
+        // purple_sys defaults extra_info.options to contain:
+        // OPT_PROTO_NO_PASSWORD | OPT_PROTO_IM_IMAGE | OPT_PROTO_CHAT_TOPIC
+        // All we have to do is remove the no_password one.
+        self.extra_info.options = purple_sys::PurpleProtocolOptions::OPT_PROTO_IM_IMAGE
```

Comment:
looking at the bindgen code, I think we can access the actual `int` value (and set it) with `.0`:

```
self.extra_info.options & PurpleProtocolOptions(!OPT_PROTO_NO_PASSWORD.0)
```

Reply:
Thanks, that looks good!

I've pushed a version of this. Do you mind looking again?